### PR TITLE
Add GenericExport

### PR DESCRIPTION
**Pull request by thenav56**
- Add export for project stats https://github.com/the-deep/server/issues/1193

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] tests
- [x] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: apps/export/mutation.py [hunk 1]
```diff
@@ -77,14 +99,53 @@ def perform_mutate(cls, root, info, **kwargs):
...
+
+class CancelUserGenericExport(UserGenericExportMutationMixin, GrapheneMutation):
+    class Arguments:
+        id = graphene.ID(required=True)
+    model = Export
+    result = graphene.Field(UserGenericExportType)
+
+    @classmethod
+    def check_permissions(cls, *args, **_):
+        return True  # Allow all to create calcel their exports
```
Comment (k9845) [inline apps/export/mutation.py]:
```suggestion
        return True  # Allow all to cancel their exports
```

File: apps/export/schema.py [hunk 2]
```diff
@@ -81,13 +89,50 @@ def resolve_file_download_url(root, info, **kwargs):
...
+
+
 class UserExportListType(CustomDjangoListObjectType):
     class Meta:
         model = Export
         filterset_class = ExportGQLFilterSet
 
 
-class Query:
+class ProjectQuery:
```
Comment (k9845) [inline apps/export/schema.py:135]: `ProjectExportQuery` might be suitable.

File: apps/export/serializers.py [hunk 3]
```diff
@@ -343,3 +345,57 @@ class Meta:
 
     def create(self, _):
         raise serializers.ValidationError('Not allowed using this serializer.')
+
+
+class UserGenericExportFiltersqlSerializer(serializers.Serializer):
```
Comment (k9845) [inline apps/export/serializers.py]:
```suggestion
class UserGenericExportFiltersGqlSerializer(serializers.Serializer):
```

File: apps/export/tasks/tasks_projects.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            'Project Stakeholders': get_organizations_display(project),
+            'Geo Areas': ','.join(regions_qs.values_list('title', flat=True).distinct()),
+            'Analysis Framework': project.analysis_framework_title,
+            'Description': project.description,
+            'Status': project.status,
+            'Test project (Y/N)': 'Y' if project.is_test else 'N',
+            'Members Count': members_qs.count(),
+            'Sources Count': leads_qs.count(),
+            'Entries Count': entries_qs.count(),
+            '# of Exports': exports_qs.count(),
```
Comment (bimal125) [inline apps/export/tasks/tasks_projects.py:108]: Can't we use a better name instead of `# of Exports`?

**Review by k9845 — APPROVED** [review]

**Review by bimal125 — APPROVED** [review]

File: apps/export/tasks/tasks_projects.py (hunk 4 above)
Comment (thenav56) [reply apps/export/tasks/tasks_projects.py:108]: From https://docs.google.com/spreadsheets/d/1XVNEClMhwDWpeAPN-dwSZ1xJZGfdqxSsUGzHrlNARmM/edit#gid=238466220&range=Q1

**Pull request merged**